doc/latex.rst

```diff
@@ -527,7 +538,6 @@ Keys that are set by other options and therefore should not be overridden are:
 ``'title'``
 ``'release'``
 ``'author'``
-``'makeindex'``
```

Comment:
I have searched througout our files and I see no reason why this ``'makeindex'`` was said to be set by other options. There may have been some confusion with the ``makeindex`` step which uses ``mendex`` with Japanese class, or it was right in the past, but something changed.